### ISSUE TITLE
IE 8

### ISSUE THREAD

**Issue by dcmwong** [nhsevidence/NICE.TopHat#22]
The keyboard navigation stuff doesn't work in IE8

- [x] Test in IE8
- [X] Run in BrowserStack
- [x] finish build trigger not working
- [x] docker errors not bubbling to team city errors
- [x] functional tests erroring as pointing to old file not wildcard folder
- [x] should delete dist folder and ignore from git
- [x] first step would need to build artifact to dist folder (and ideally use artifact snapshot so we only build once) to share across build steps
- [x] add snapshot dependency/dependencies in third build step

**Comment by dcmwong** [comment]
Done
